styles.css

```diff
@@ -0,0 +1,125 @@
+html {
+    box-sizing: border-box; 
+  }
+  
+* {
+    box-sizing: inherit;
+    margin: 0;
+  }
+body {
+     margin: 0;
+}
+header {
+    display: flex;
+    flex-wrap: wrap;
+    height: 30vh;
+    justify-content: space-between;
+    align-content: center;
+    background: rgb(2,0,36);
+    background: linear-gradient(90deg, rgb(5, 2, 65) 0%, rgb(40, 30, 128) 57%, rgba(0,212,255,1) 100%);
+    color: ghostwhite;
+    Font-Family: 'Roboto Condensed', Sans-Serif;
+    Font-Size: 2rem;
+    
+}
+h1 {
+    align-self: center;
+    justify-self: flex-start;
+    font-weight: bolder;
+    margin-left: 5%;
+}
+nav {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+    margin:7%;
+    font-weight: 800;
+    width:10%;
+    flex-wrap: wrap;
+
+
+    
+}
+a {
+    text-decoration: none;
```

Comment:
Usually in the web links are displayed with an underline. I would add a `border-bottom` if you decide to remove it.

Reply:
Still undecided how to style the web links, but I will consider it.

Reply:
Coolio! And well done! 😎

Reply:
Thank you 😸